x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.ts

```diff
@@ -118,90 +118,60 @@ export const buildMatchAny = ({
 };
 
 export const buildNested = ({
-  item,
+  entry,
   language,
 }: {
-  item: EntryNested;
+  entry: EntryNested;
   language: Language;
 }): string => {
-  const { field, entries } = item;
+  const { field, entries: subentries } = entry;
```

Comment:
I like the `subentries` will look to use that in future PRs for clarity on what it refers to.